.github/workflows/IntegrationTest.yml

```
@@ -0,0 +1,55 @@
+name: IntegrationTest
+on:
+  push:
+    branches: [master]
+    tags: [v*]
+  pull_request
```

```suggestion
  pull_request:
```
@mattBrzezinski  the `:` was needed else i get this error
https://github.com/JuliaDiff/ChainRulesTestUtils.jl/actions/runs/488680037

---

Oh strange!